.travis.yml

```diff
@@ -0,0 +1,32 @@
+# This is a common .travis.yml for generating library release zip files for
+# CircuitPython library releases using circuitpython-build-tools.
+# See https://github.com/adafruit/circuitpython-build-tools for detailed setup
+# instructions.
+
+dist: trusty
+sudo: false
+language: python
+python:
+    - "3.6"
+
+cache:
+    pip: true
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_TOKEN
+  file_glob: true
+  file: $TRAVIS_BUILD_DIR/bundles/*
+  skip_cleanup: true
+  overwrite: true
+  on:
+    tags: true
+
+install:
+  - pip install pylint circuitpython-build-tools Sphinx sphinx-rtd-theme
+
```

Comment:
Could you change this to:
```
- pip install circuitpython-build-tools Sphinx sphinx-rtd-theme
- pip install --force-reinstall pylint==1.9.2
```
While pylint is currently happy, who knows what will get added in future versions. :smile: